ballerina/tests/resources/proto-files/data-types/helloWorldWithNestedMessage.proto

```diff
@@ -18,6 +18,8 @@
 
 syntax = "proto3";
 
+package choreo.apis;
```

Comment:
shall we use a different package name. some random name

Reply:
Updated in 527e805